h/static/scripts/login-forms/components/LoginForm.tsx

```diff
@@ -71,6 +71,9 @@ export default function LoginForm() {
           </Button>
         </div>
       </Form>
+      {config.features.log_in_with_orcid && (
+        <a href={routes.loginWithORCID}>Continue with ORCID</a>
+      )}
```

Comment:
Just a plain **Continue with ORCID** link added to the login page for now. This isn't intended to be the final design.